examples/eventsprofiles_btyd.py

```diff
@@ -23,79 +25,82 @@
 START_TOKEN_DISCR = "StartToken"
 COLS = eventsprofiles_datamodel.target_cols + eventsprofiles_datamodel.cont_feat + eventsprofiles_datamodel.discr_feat
 
-data_dir = str(Path(__file__).parent.absolute())
+if __name__ == "__main__":
+    pl.seed_everything(9473)
```

Comment:
Any reason for this seed? Is there an easy way to remember it?

Reply:
it spells WISE :)